pkg/machinery/go.mod

```diff
@@ -7,16 +7,16 @@ go 1.20
 replace gopkg.in/yaml.v3 => github.com/unix4ever/yaml v0.0.0-20220527175918-f17b0f05cf2c
 
 require (
-	github.com/containerd/go-cni v1.1.8
-	github.com/cosi-project/runtime v0.3.0-alpha.7
+	github.com/containerd/go-cni v1.1.9
+	github.com/cosi-project/runtime v0.3.0-alpha.8
 	github.com/dustin/go-humanize v1.0.1
 	github.com/evanphx/json-patch v5.6.0+incompatible
 	github.com/ghodss/yaml v1.0.0
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/hashicorp/go-version v1.6.0
 	github.com/jsimonetti/rtnetlink v1.3.1
 	github.com/mdlayher/ethtool v0.0.0-20221212131811-ba3b4bc2e02c
-	github.com/opencontainers/runtime-spec v1.0.3-0.20210326190908-1c3f411f0417
+	github.com/opencontainers/runtime-spec v1.1.0-rc.1.0.20230306064114-a7c7692b2a2c
```

Comment:
let's not update this please, we had it discussed last week

Reply:
yeah, sorry forgot about it